docs/user-guide/release-notes/2.14.x.rst

```diff
@@ -0,0 +1,30 @@
+=======================
+Pulp 2.14 Release Notes
+=======================
+
+Pulp 2.14.0
+===========
+
+New Features
+------------
+
+* An additional `header` property has been added to the alternate content source descriptor.
+  This allows users to specify additional HTTP headers when using an alternate content source.
+
+
+Bug Fixes
+---------
+
+See the list of :fixedbugs_pulp:`2.14.0`
+
+Upgrade instructions
```

Comment:
The "normal process" upgrade instructions have been in every release note 2.11+
Personally I think it has its merits since when the new version comes out the user does not have to dig for upgrade instructions.